apps/sendsignal.go

```diff
@@ -0,0 +1,63 @@
+package apps
+
+import (
+	"context"
+
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/io"
+	"github.com/Scalingo/go-scalingo/v6"
+	"github.com/Scalingo/go-utils/errors"
+)
+
+func keepUniqueContainersWithNames(containers []scalingo.Container, names []string) map[string]scalingo.Container {
+	containersToKill := map[string]scalingo.Container{}
+
+	for _, name := range names {
+		hasMatched := false
+		for _, container := range containers {
+			if container.Label == name {
+				containersToKill[container.ID] = container
+				hasMatched = true
+			}
+		}
+		if !hasMatched {
```

Comment:
I don't think you need `hasMatched`
```suggestion
		for _, container := range containers {
			if container.Label == name {
				containersToKill[name] = container
			}
		}
		if _,ok := containersToKill[name]; !ok {
```